shigatyper/2.0.5/Dockerfile

```diff
@@ -0,0 +1,143 @@
+  rm -v samtools-${SAMTOOLSVER}.tar.bz2 && \
+  cd samtools-${SAMTOOLSVER} && \
+  ./configure && \
+  make && \
+  make install 
+
+# shigatyper depends on bcftools
+RUN wget https://github.com/samtools/bcftools/releases/download/${BCFTOOLSVER}/bcftools-${BCFTOOLSVER}.tar.bz2 && \
+  tar -xjf bcftools-${BCFTOOLSVER}.tar.bz2 && \
+  rm -v bcftools-${BCFTOOLSVER}.tar.bz2 && \
+  cd bcftools-${BCFTOOLSVER} && \
+  make && \
+  make install
+
+# shigatyper depends on minimap2
+RUN curl -L https://github.com/lh3/minimap2/releases/download/v${MINIMAP2_VER}/minimap2-${MINIMAP2_VER}_x64-linux.tar.bz2 | tar -jxf -
+
+# set the environment
+ENV PATH="${PATH}:/minimap2-${MINIMAP2_VER}_x64-linux" \
+  LC_ALL=C
+
+### start of app stage ###
+# using fresh ubuntu:jammy image as base for app stage
+FROM ubuntu:jammy as app
+
+# re-instantiate thes variables
+ARG SHIGATYPER_VER
+ARG MINIMAP2_VER
+
+# LABELS must be contained in `app` stage to pass the GHActions checks
+LABEL base.image="ubuntu:jammy"
+LABEL dockerfile.version="1"
+LABEL software="shigatyper"
+LABEL software.version="${SHIGATYPER_VER}"
+LABEL description="Determine Shigella serotype using Illumina (single or paired-end) or Oxford Nanopore reads!"
+LABEL website="https://github.com/CFSAN-Biostatistics/shigatyper"
+LABEL license="https://github.com/CFSAN-Biostatistics/shigatyper/blob/master/LICENSE"
+LABEL maintainer="John Arnn"
+LABEL maintainer.email="jarnn@utah.gov"
+LABEL maintainer2="Curtis Kapsak"
+LABEL maintainer2.email="curtis.kapsak@theiagen.com"
+
+# copy in samtools and bcftools executables into app stage
+COPY --from=builder /usr/local/bin/* /usr/local/bin
+
+# copy in minimap2 executable into app stage
+COPY --from=builder /minimap2-${MINIMAP2_VER}_x64-linux /usr/local/bin
+
+# ensuring samtools & bcftools dependencies are present (dependencies for running tools, not for compiling/installing tools)
+RUN apt-get update && apt-get install --no-install-recommends -y \
+    perl \
+    zlib1g \
+    libncurses5 \
+    bzip2 \
+    liblzma-dev \
+    libcurl4-gnutls-dev \
+    python3 \
+    python3-setuptools \ 
+    wget \
+    ca-certificates \
+    && apt-get autoclean && rm -rf /var/lib/apt/lists/*
+
+# get shigatyper code; install; index fasta file
+# FYI - samtools faidx cmd has a * in the path due to version inconsistencies
+RUN wget https://github.com/CFSAN-Biostatistics/shigatyper/archive/refs/tags/${SHIGATYPER_VER}.tar.gz && \
+  tar -xf ${SHIGATYPER_VER}.tar.gz && \
+  rm -v ${SHIGATYPER_VER}.tar.gz && \
+  cd shigatyper-${SHIGATYPER_VER} && \
+  python3 setup.py install && \
+  samtools faidx /usr/local/lib/python3.10/dist-packages/ShigaTyper-*-py3.10.egg/resources/ShigellaRef5.fasta
+
+# ensuring final working directory is /data
+WORKDIR /data
+
+### start of test stage ###
+# using app stage as base image for test stage
+FROM app as test
+
+# default command is to pull up help options; can be overridden of course
+CMD ["shigatyper", "--help"]
```

Comment:
Can you move your 'CMD' action to the app layer? The test layer doesn't get deployed to dockerhub or quay.